q2_types/feature_data/tests/test_transformer.py

```diff
@@ -556,6 +606,55 @@ def test_skbio_tabular_msa_to_aligned_dna_fasta_format(self):
         for act, exp in zip(obs, input):
             self.assertEqual(act, exp)
 
+    def test_dnafasta_format_to_series(self):
+        filename = 'dna-sequences.fasta'
+        _, obs = self.transform_format(DNAFASTAFormat, pd.Series, filename)
+
+        seqs_fp = self.get_data_path(filename)
+        exp = skbio.read(seqs_fp, format='fasta', constructor=skbio.DNA)
+
+        for expected, observed in zip(exp, obs):
+            self.assertEqual(expected, observed)
+
+    def test_dnafasta_format_to_metadata(self):
+        filename = 'dna-sequences.fasta'
+        _, obs = self.transform_format(DNAFASTAFormat, qiime2.Metadata,
+                                       filename)
+
+        self.assertEqual({'SEQUENCE1', 'SEQUENCE2'}, obs.ids())
+
+        seqs_fp = self.get_data_path(filename)
+        exp = skbio.read(seqs_fp, format='fasta', constructor=skbio.DNA)
+        obs_category = obs.get_category('Sequence').to_series()
+
+        for expected, observed in zip(exp, obs_category):
+            self.assertEqual(expected, observed)
+
+    def test_aligned_dnafasta_format_to_series(self):
+        filename = 'aligned-dna-sequences.fasta'
+        _, obs = self.transform_format(AlignedDNAFASTAFormat, pd.Series,
+                                       filename)
+
+        seqs_fp = self.get_data_path(filename)
+        exp = skbio.read(seqs_fp, format='fasta', constructor=skbio.DNA)
```

Comment:
Removing this test as part of the [request above](https://github.com/qiime2/q2-types/pull/125/files#r121547148).